.github/workflows/main.yml

```diff
@@ -1,6 +1,9 @@
-name: Test and Publish sdist tarball to TestPyPi
+name: Test and Publish sdist tarball to PyPi
 
 on:
+  schedule:
+    # Runs every sunday at 3 a.m.
+    - cron: '0 3 * * SUN'
```

Comment:
Do we have to specify Sunday? Just once a week would be good.

Reply:
We can specify a day of the week (MON-SUN) or a day of the month (1-31) or no day at all (-> runs each day). Setting it to Sunday seemed like a decent way to make it run once a week.

Reply:
Ok